src/Services/test/CloudFoundry/CloudFoundryServiceTests.cs

```diff
@@ -1389,23 +1389,26 @@ public async Task DeployAppAsync_ReturnsFalseResult_WhenCfPushCommandFails()
 
             _mockFileLocatorService.Setup(mock => mock.DirContainsFiles(It.IsAny<string>())).Returns(true);
 
-            DetailedResult result = await _sut.DeployAppAsync(FakeCfInstance, FakeOrg, FakeSpace, FakeApp.AppName, _fakeProjectPath, _defaultFullFWFlag, stdOutCallback: null, stdErrCallback: null, manifestPath: null);
+            DetailedResult result = await _sut.DeployAppAsync(FakeCfInstance, FakeOrg, FakeSpace, FakeApp.AppName, _fakeProjectPath, _defaultFullFWFlag, stdOutCallback: null, stdErrCallback: null, stack: null, manifestPath: null);
```

Comment:
A sample advantage of the converting to a single arg would be simpler test setup.  You could have a mock object for the struct and reuse/tweak as needed.